.github/workflows/CI.yaml

```diff
@@ -29,12 +29,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.9", "3.10"]
-        rdkit-version: ["2021", "latest"]
+        python-version: ["3.9", "3.10", "3.11"]
+        rdkit-version: ["latest", ]
```

Comment:
Can we keep this?

Reply:
It's now in the "include" section (rather than having several excludes)